supabase/migrations/005_notifications.sql

```
@@ -0,0 +1,103 @@
+        FOREIGN KEY (actor_id) REFERENCES profiles(id) ON DELETE CASCADE,
+    CONSTRAINT notifications_post_id_fkey
+        FOREIGN KEY (post_id) REFERENCES posts(id) ON DELETE CASCADE
+);
+
+CREATE INDEX notifications_recipient_id_idx ON notifications(recipient_id);
+CREATE INDEX notifications_created_at_idx ON notifications(created_at DESC);
+
+-- RLS
+ALTER TABLE notifications ENABLE ROW LEVEL SECURITY;
+
+CREATE POLICY "Users can view own notifications"
```

SQLFluff (4.1.0)

[error] Do not use special characters in identifiers.

(RF05)